Habinezajanvier/my-brand-api
# #8 setup testing environment

#### What does this PR do?
Set up a testing environment
#### Description of Task to be completed?
- Installing jest for testing
- Configuring jest 
- Installing supertest
#### How should this be manually tested?
- Clone this repo `git clone https://github.com/Habinezajanvier/my-brand-api.git`
- Switch your working directory `cd my-brand-api`
- Checkout the branch `git checkout ch-setup-testing-env-#8`
- Create a `.env` file and update it with data from `.env-sample`
- Install all dependencies `npm install`
- In your terminal run `npm test` 
> You should see jest running a test cases and passing successfully
#### Any background context you want to provide?
- NodeJS should be installed on your PC
- MongoDB should be installed in your PC
#### What are the relevant pivotal tracker stories?
Click [here](https://trello.com/c/faYtgAKH/8-setting-a-testing-environment) for relevant **Trello** story
